dev/license/updatelicense.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/node
```

Comment:
It's not working on my macbook. Only `#!/usr/bin/env node` is working correctly.